src/test/java/org/andersen/service/impl/HotelServiceImplTests.java

```diff
@@ -0,0 +1,53 @@
+package org.andersen.service.impl;
+
+import org.andersen.model.Apartment;
+import org.andersen.model.Hotel;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class HotelServiceImplTests {
+
+    @Mock
+    private Hotel hotel;
```

Comment:
not needed

Reply:
Cannot invoke "org.andersen.model.Hotel.getApartments()" because "this.hotel" is null

without @Mock

Reply:
yes)  you should create new hotel in beforeEach method ))
mock we use for the objects, like a dependency in your target class 

ok.. I miss that its a dependency)) sorry) its OK)) leave as it is))